src/Model/Entry.php

```diff
@@ -36,6 +36,7 @@ class Entry implements EntryRequiredInterface
     private string $title;
 
     private \DateTimeInterface $updated;
+    private Author $author;
```

Comment:
[RFC 4287](https://tools.ietf.org/html/rfc4287#section-4.1.1) Entry elements may contain more than 1 author. An array or collection of Authors is needed